### PR TITLE
'Leaky_Relu' op suuport

### LINKED ISSUE #62: Operator request: Leaky Relu (Unsupported TFLite OP: 98 LEAKY_RELU!)
**Note**: Currently, we only accept [TensorFlow Lite builtin operators](https://jackwish.net/tflite/docs/BuiltinOperator.m.html) request.

**What is the TensorFlow Lite operator you need**
(Please attach a tiny TFLite model `*.tflite`, which contains the operator you need ONLY.)
LeakyRelu

**What kind of service you are trying to deploy your model?**
.tflite to .onnx

**Would you like to contribute the operator?**

**Additional context**

### PR DESCRIPTION
**Pull request by shilpan97**
hey, wasn't sure if my changes are correct(please treat as initial attempt). 
When trying to run on my machine I got the following error message:
AttributeError: type object 'ActivationFunctionType' has no attribute 'LEAKY_RELU'. 
Not sure how to fix it. 

Please feel free to edit directly. 
Thanks!

### CODE REVIEW

**Comment by shilpan97** [comment]
formula:
f(x) = alpha * x for x < 0 , f(x) = x for x >= 0

**Review by zhenhuaw-me — CHANGES_REQUESTED** [review]
@shilpan97 Thanks for contributing to the project! I am very pleased that we received a new contribution after the last release.

I left several comments in the PR. In general (just like what we mentioned in the [how to enable new operator tutorial](https://github.com/jackwish/tflite2onnx/blob/master/docs/how-to-enable-new-operator.md)), we'd like to attach the test (including the TFLite model which contains the operator only if possible, and the test code) in our code change, such that we know what functionality is working and what needs to be fixed whenever we make a change.

This can be done since we have written a detailed guide of enabling operators. Please feel free to raise any question about the document or the enabling.

Thanks!

**Review by shilpan97 — COMMENTED** [review]
Changes based on previous feedback. I don't think I can give my .tflite to you unfortunately due to permissions. Hopefully this changes fix it. Thank you! Also, can you let me know when are you pushing your next release thanks. Earlier would be great.

**Review by shilpan97 — COMMENTED** [review]
removed from act_type

File: tflite2onnx/op/activation.py [hunk 1]
```diff
@@ -68,6 +69,11 @@ def parse(self):
             alpha = self.parseInput(1)
             alpha.shape.insert(0, 1)
 
+        if opcode == tflite.BuiltinOperator.LEAKY_RELU:
+            op_opt = op.BuiltinOptions()
+            option = tflite.LeakyReluOptions()
+            option.Init(op_opt.Bytes, op_opt.Pos)
+            alpha = option.Alpha()
```
Comment (jackwish) [inline tflite2onnx/op/activation.py:76]: The alpha needs to be passed to the ONNX operator. The code here actually does nothing. The `Concat` code may tip you how to do it.

**Review by zhenhuaw-me — CHANGES_REQUESTED** [review]
Thanks for the update! By ***testing*** I mean create a new TFLite model which contains the operator (`LeakyReLU` for this case) only, so there is no legal issue. The guide includes every step to do this, I believe you will have a better understanding of this tool if read through it.

**Comment by jackwish** [comment]
@shilpan97 You can take a look at this PR https://github.com/jackwish/tflite2onnx/pull/25/files which implements the `Prelu` which should be very similar to your work.

**Pull request closed without merging**